backend/src/accounts.ts

```diff
@@ -117,115 +117,40 @@ function ignoreIfDoesNotExist(error: unknown): null {
-      lockupAccountId,
-      "get_staking_pool_account_id",
-      {}
-    ).catch(ignoreIfDoesNotExist),
-    sendJsonRpc("EXPERIMENTAL_protocol_config", { finality: "final" }),
+    getLockupAccountId(accountId),
   ]);
 
   if (accountInfo === null) {
     return null;
   }
 
-  const storageUsage = new BN(accountInfo.storage_usage);
-  const storageAmountPerByte = new BN(
-    protocolConfig.runtime_config.storage_amount_per_byte
-  );
-  const stakedBalance = new BN(accountInfo.locked);
-  const nonStakedBalance = new BN(accountInfo.amount);
-  const minimumBalance = storageAmountPerByte.mul(storageUsage);
-  const availableBalance = nonStakedBalance
-    .add(stakedBalance)
-    .sub(BN.max(stakedBalance, minimumBalance));
-
-  let lockupDelegatedToStakingPoolBalance: BN | null = null;
-  if (lockupStakingPoolAccountId) {
-    lockupDelegatedToStakingPoolBalance = await callViewMethod<string>(
-      lockupStakingPoolAccountId,
-      "get_account_total_balance",
-      {
-        account_id: lockupAccountId,
-      }
-    )
-      .then((balance) => new BN(balance))
-      .catch(ignoreIfDoesNotExist);
-  }
-
-  let totalBalance = stakedBalance.add(nonStakedBalance);
-  let lockupDetails: {
-    accountId: string;
-    totalBalance: string;
-    lockedBalance: string;
-    unlockedBalance: string;
-  } | null = null;
-  // The following section could be compressed into more complicated checks,
-  // but it is left in a readable form.
-  if (accountId === lockupAccountId) {
-    // It is a lockup account
-    if (lockupDelegatedToStakingPoolBalance) {
-      totalBalance.iadd(lockupDelegatedToStakingPoolBalance);
-    }
-  } else {
-    // TODO: could it be that `lockupLockedBalance` is null but we still have info?
-    if (lockupAccountInfo && lockupLockedBalance) {
-      // It is a regular account with lockup
-      const lockupStakedBalance = new BN(lockupAccountInfo.locked);
-      const lockupNonStakedBalance = new BN(lockupAccountInfo.amount);
-      let lockupTotalBalance = lockupStakedBalance.add(lockupNonStakedBalance);
-      if (lockupDelegatedToStakingPoolBalance) {
-        lockupTotalBalance.iadd(lockupDelegatedToStakingPoolBalance);
-      }
-      totalBalance.iadd(lockupTotalBalance);
-      lockupDetails = {
-        accountId: lockupAccountId,
-        totalBalance: lockupTotalBalance.toString(),
-        lockedBalance: lockupLockedBalance.toString(),
-        unlockedBalance: lockupTotalBalance.sub(lockupLockedBalance).toString(),
-      };
-    }
-    // It is a regular account without lockup
-  }
-
   return {
-    storageUsage: storageUsage.toString(),
-    stakedBalance: stakedBalance.toString(),
-    nonStakedBalance: nonStakedBalance.toString(),
-    minimumBalance: minimumBalance.toString(),
-    availableBalance: availableBalance.toString(),
-    totalBalance: totalBalance.toString(),
-    lockupAccountId: lockupDetails?.accountId,
-    lockupTotalBalance: lockupDetails?.totalBalance,
```

Comment:
@frol
Why did we use to have this much of lockup data?
Also minimum balance, available balance and total balance were not used at all.

Reply:
@luixo Historically, we tried to compute "total account balance", but we failed miserably on that side, but considered to get back to it with a similar approach as Wallet did for the profile page: https://wallet.near.org/profile/frol.near